### ISSUE TITLE
renaming third party code/typestubs

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pyright#643]
**Describe the bug**
rename sometime changes third party or typestub code

**To Reproduce**
a.py
```python
from testLib1 import Test1 as t1

a = t1()
```
testLib1/__init__.py
```python
class Test1:
    pass
```

rename t1 to t2.

**Expected behavior**
only t1 is renamed to t2.

but Test1 is also renamed to t2.

**VS Code extension or command-line**
latest pyright on vscode extension

**Additional context**
renaming t1 to t2 affecting Test1 is also a bug. but Test1 ever changed by rename seems to be another bug. rename should not ever change code in third party library or typestub.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now addressed in version 1.1.35, which I just published.
